Sources/Deferred.swift

```diff
@@ -28,9 +28,15 @@ private struct DispatchBlockMarker: CallbacksList {
         // Executing the block "unblocks" it, calling all the `_notify` blocks
         block()
     }
-    
-    func notify(upon queue: dispatch_queue_t, body: dispatch_block_t) {
-        dispatch_block_notify(block, queue, body)
+
+    func notify(upon executor: ExecutorType, body: dispatch_block_t) {
```

Comment:
"Notify upon queue" made sense. I'm not sure "notify upon executor" does. I think `notify(executor: …` makes more sense; since the argument is an `Executor`, I don't know that a preposition is even needed. It's like "go tell the executor…".

The "body" argument name doesn't make much sense in either case I think. :\